server/data.js

```diff
@@ -0,0 +1,184 @@
+let database = {
```

Comment:
I noticed that if you don't input any words into the search, the buttons wont work but I think it'd be pretty cool if the random one still worked. Also if it showed an error when other words are being typed.

Reply:
Ah yeah that was a last min fix in the client side folder. Origionally it loaded the rabbits page on an empty seach which I only wanted to be on a non cat, rabbit, ostritch search. I will try make it so the I'm lucky btn still works. Thank you

Reply:
but... we like rabbits 😢

Reply:
Nothing wrong with rabbits 😢 , it was just an idea